addon/ng2/blueprints/ng2/files/angular-cli.json

```diff
@@ -5,9 +5,24 @@
   },
   "apps": [
```

Comment:
This PR isn't aimed to implement the multiple apps workflow yet, so it's better not to change it.